pallets/confidential/Cargo.toml

```diff
@@ -30,7 +30,7 @@ sp-runtime-interface = { git = "https://github.com/paritytech/substrate", defaul
 # Crypto
 rand_core = { version = "0.5", default-features = false }
 rand = { version = "0.7.3", default-features = false, optional = true }
-cryptography = { git = "https://github.com/PolymathNetwork/cryptography.git", branch = "develop", default-features = false }
+cryptography = { git = "https://github.com/PolymathNetwork/cryptography.git", branch = "CRYP-166/take_in_slices", default-features = false }
```

Comment:
I will be reverting back to `develop` as soon as my other PR goes in.

Reply:
LGTM but I'd suggest merging this after the PR in crypto repo is merged.

Reply:
I'll make sure that happens.